### PR TITLE
Enable testing data

### PR DESCRIPTION
**Pull request by shuklaayush**
**Description of PR**
Uncomment testing data code and fix infinite loop.

**Type of PR**

- [x] Bugfix
- [ ] New feature

**Checklist**

- [x] Compiles and passes lint tests
- [x] Properly formatted
- [x] Tested on desktop
- [x] Tested on phone

### CODE REVIEW

**Comment by huzefa-amura** [comment]
@shuklaayush : Hey a good one !!!

Can you check the mobile view? The alignment seems to be a little disturbed.

File: src/components/mapexplorer.js [hunk 1]
```diff
@@ -393,16 +392,9 @@ export default function ({
...
-    stateTestData.forEach((testObj, index) => {
-      if (testObj.state === panelRegion.name) {
-        found = true;
-        setTestObj(testObj);
-      }
-    });
-    if (!found) setTestObj({});
-  }, [panelRegion, stateTestData, testObj]);*/
+  useEffect(() => {
+    setTestObj(stateTestData.find((obj) => obj.state === panelRegion.name));
```
Comment (bee-rickey) [inline src/components/mapexplorer.js:396]: This takes the first entry in the list. We should take the last entry in the list (test data is available per state per date basis in an ascending order of dates).

Comment (shuklaayush) [reply src/components/mapexplorer.js:396]: I didn't bother to look at the format. I've reversed the data when it's loaded. Hope that's fine

**Comment by shuklaayush** [comment]
@huzefa-amura I'll leave that to @jeremyphilemon. I don't do design

**Pull request merged**